### PR TITLE
Fixes Issue No #46 , #59 , #109 , #54 | Adds Speaker Profile Info | Adds Social Media Details | Increases Sizes

### PR DESCRIPTION
**Pull request by ashutoshsaboo**
Solves & Fixes Issue #46 , #54 , #59 and #109 . 

1. Adds Speaker Profile Info
2. Adds Social Media Details
3. Increases Social Media Size Buttons
4. Photo added in detail view of speakers
5. Links and Social Channels given now
6. Session of speakers added - In Speaker Matthias Spielkamp's detail view
7. Organisation Link also Added
8. Speakers Overview Doesn't start with a `comma`
8. Other details required in the respective issues are also added

@mariobehling Please have a look at this PR.

Please suggest in case if any other changes are required to improve the code. 

Solves & Fixes Issue #46 , #59 , #109 and #54 .

Thanks! :smile:

### CODE REVIEW

**Comment by ashutoshsaboo** [comment]
@mariobehling @creativepsyco Is this fine? Is this PR ready to be merged?

**Comment by ashutoshsaboo** [comment]
@jig08 Could you review this PR as well? Thanks! :smile:

**Comment by ashutoshsaboo** [comment]
@SaptakS @joerx @daminisatya @mariobehling @jig08 @AmayAgrawal @championswimmer @creativepsyco  Could someone review the PR, please! Thanks! :smiley:

**Comment by ashutoshsaboo** [comment]
Can the issues #46 and #54 be closed, and this PR can be merged?

File: index.html~ [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```
Comment (championswimmer) [inline index.html~]: Remove

**Comment by ashutoshsaboo** [comment]
@championswimmer Hi. So, I have made the required changes. So, can this be merged now?

@joerx @mariobehling @jig08 @AmayAgrawal @creativepsyco @SaptakS @daminisatya 

Thanks! :smile:

**Pull request closed without merging**